jaxrs/pom.xml

```diff
@@ -123,6 +123,18 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.reactiverse</groupId>
+            <artifactId>reactive-contexts-propagators-rxjava1</artifactId>
+            <version>${reactiversecontexts.version}</version>
+            <exclusions>
+                <exclusion>
```

Comment:
Will there be a risk of not getting the version of rxjava needed to support contexts?

Reply:
Good point. I don't believe it will (since the used api is available in the version we use, 1.2.10). However, since we're using 1.3.8 in our internal modules perhaps we should update the version here as well?